index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CompoundCode - door to door programming lessons.</title>
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Quicksand:wght@700&display=swap">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&family=Lato:wght@300&display=swap">
+  <link rel="stylesheet" href="css/index.css">
+</head>
+<body>
+  <aside class="desktop-only d-flex row top-mini-social-actions">
+    <a href="#"><img src="images/facebook-32.png" alt="Like us on Facebook"></a>
+    <a href="https://twitter.com/mwanzabj"><img src="images/twitter-32.png" alt="Follow us on Twitter"></a>
+    <a href="#">English</a>
+    <a href="hhtps://facebook.com/CompoundCode">My Page</a>
+    <a href="#">Logout</a>
+  </aside>
+  <header class="d-flex cols wrap top-main-header is-mobile">
+    <a href="#" class="mobile-nav-trigger mobile-only" title=""><img src="images/menu-trigger-1.png" alt="Menu"></a>
+    <a href="#" class="desktop-only"><img src="images/compoundcode-logo-1.png" alt="Home"></a>
+    <nav class="nav-menu is-mobile d-flex cols">
+        <a href="#" class="close-nav-trigger mobile-only" title="">
+          <img src="images/close-menu.png" alt="Menu"> 
+        </a>
+        <a href="about.html" class="active wrap">About</a>
+        <a href="#">Program</a>
+        <a href="#">Join</a>
+        <a href="#">Sponsor</a>
+        <a href="#">News</a>
+        <a href="#" class="active-border">CC Campaign</a>
+    </nav>
+  </header>
```

Comment:
- [ ] For mobile size Navbar's background should be transparent or the same color/pattern as the top section background. Currently, you have a white background, you can check the design below. Let's fix that.

![Screenshot from 2021-10-14 08-03-09](https://user-images.githubusercontent.com/40503704/137268295-5beb31c1-da87-417f-a8d3-615924d27073.png)

![Screenshot from 2021-10-14 08-07-37](https://user-images.githubusercontent.com/40503704/137268483-67cd9a2b-6452-4f06-8dea-563ca2c7ae2e.png)

Reply:
✅  Done and fixed as guided 🤓